packages/executor/src/services/BundlingService.ts

```diff
@@ -148,8 +148,7 @@ export class BundlingService {
           // eslint-disable-next-line @typescript-eslint/no-unused-vars
           const { gasLimit, ...txWithoutGasLimit } = tx;
           // some chains, like Bifrost, don't allow setting gasLimit in estimateGas
-          const estimatedGasLimit = await wallet.estimateGas(txWithoutGasLimit);
-          tx.gasLimit = estimatedGasLimit;
+          await wallet.estimateGas(txWithoutGasLimit);
```

Comment:
Why have we removed the gasLimit check here? I thought that was affecting Gnosis/Bifrost chain?

Reply:
the estimated gasLimit is too low. There's a check in the EntryPoint contract that requires setting a higher gas limit. The purpose of calling `wallet.estimateGas` here is too make sure that the transaction does not revert